colorbleed/plugins/maya/publish/collect_renderlayers.py

```diff
@@ -32,7 +32,12 @@ def process(self, context):
         context.data["startFrame"] = start_frame
         context.data["endFrame"] = end_frame
 
-        # Get render layers
+        # Get all linked renderlayers
```

Comment:
Shall we add another comment line here?
```python
    # Get all valid renderlayers
    # This is how Maya populates the renderlayer display
```

Since it's such a weird bit of code I want to emphasize that its based on some code from how Maya does it.